concepts/range-iteration/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Go has a special \"for range\" loop to easily iterate over collections of data.",
```

Comment:
I believe range is formally a function, so let's say:

Go has a special range function to easily iterate over collections of data.

Reply:
range [is a keyword](https://golang.org/ref/spec#Keywords), so it's not accurate to call it a function. But I don't think mentioning it as being keyword is useful either - I think referring it as a "for range" loop or "range form of the for loop" is descriptive enough.

Reply:
SG, its behavior just acts like a function from my perspective. Resolving this comment sicne this works.